db/migrate/20170509224500_remove_agent_name_from_jobs.rb

```diff
@@ -1,5 +1,6 @@
 class RemoveAgentNameFromJobs < ActiveRecord::Migration[5.0]
   def change
+    return if previously_migrated_as?("20170321171945")
```

Comment:
@yrudman Please remove this.  I'd rather save the previously_migrated_as? for stuff that's already been released.